.gitignore

```diff
@@ -1,3 +1,8 @@
 # Logs, cache, and temp files
 *.log
 .DS_Store
+
+
+# Added by cargo
+
+/target
```

Comment:
There shouldn't be any changes to this file. Revert is by running `git checkout master -- .gitignore`